eap-xp/runtime-image/test/Dockerfile

```diff
@@ -0,0 +1,7 @@
+#Be sure to update eap-testapp with the name of the image generated with s2i.
+FROM jboss-eap-7/eap-xp10-openjdk11-runtime-openshift-rhel8
```

Comment:
name of the image is not correct as we changed it recently, it should be` jboss-eap-7/eap-xp1-openjdk11-runtime-openshift-rhel8`